src/tests/exchange/test_liquidation_handler.cairo

```diff
@@ -34,6 +50,42 @@ fn test_exec_liquidation_true() {
         );
 }
 
+#[test]
+#[ignore]
+fn test_create_liquidation_order() {
+    let collateral_token: ContractAddress = contract_address_const::<1>();
+    let (
+        data_store, liquidation_keeper, liquidation_handler_address, liquidation_handler_dispatcher
+    ) =
+        _setup();
+    start_prank(liquidation_handler_address, liquidation_keeper);
+    let account = contract_address_const::<'account'>();
+    let market = contract_address_const::<'market'>();
+    let is_long = true;
+    let key: felt252 = get_position_key(account, market, collateral_token, true);
+    let mut position: Position = create_new_position(
+        key, account, market, collateral_token, is_long, position_no: 1
+    );
+
+    data_store.set_position(key, position);
+
+    let mut state_base: BaseOrderHandler::ContractState =
+        BaseOrderHandler::unsafe_new_contract_state(); //retrieve BaseOrderHandler state
+
+    let key: felt252 = create_liquidation_order(
```

Comment:
to test `create_liquidation_order` which creates an order and add it to `data_store` i have added this test. but it failing with cryptic error message (ig its coming from call to `unsafe_new_contract_state`)

```
Collected 1 test(s) and 2 test file(s)
Running 1 test(s) from satoru package
[FAIL] satoru::tests::exchange::test_liquidation_handler::test_create_liquidation_order

Failure data:
    Got an exception while executing a hint:
    Bad syscall_ptr; expected: Relocatable { segment_index: 10, offset: 0 }, got: Relocatable { segment_index: 10, offset: 60 }.

Running 0 test(s) from tests/utils/test_pricing_utils.cairo
Tests: 0 passed, 1 failed, 0 skipped

Failures:
    satoru::tests::exchange::test_liquidation_handler::test_create_liquidation_order
```

but i needed to call it to get `event_emitter` required by `create_liquidation_order` function. Any other way to get `event_emitter`?

Reply:
cc: @Sk8erboi84

Reply:
you have passed wrong arguments in the function, you should deploy an event_emitter contract in setup() and pass it to create_liquidation_order. do the same for data_store, without calling base_order_handler

Reply:
`setup` already deploys data_store, so i think i will only need to add `event_emitter`. should i make this change in `setup` function (requiring to update every call to this function) or create a new function called `setup_event_emitter`

Reply:
got it, thanks!

Reply:
You need to update setup() and pass data_store returned by setup() to create_liquidation fonction

Reply:
yeah realized that just after posting previous comment, its fixed.